src/CodeGen_LLVM.h

```diff
@@ -514,6 +517,11 @@ class CodeGen_LLVM : public IRVisitor {
      * across backends. */
     virtual void codegen_vector_reduce(const VectorReduce *op, const Expr &init);
 
+    /** Split up a VectorReduce node if possible, or generate LLVM
```

Comment:
Please elaborate on this comment, also nit: lines in the block comment should start with *